src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,9 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+
+    @Override
+    public int compareTo(Person o) {
+        return (this.getName().toString()).compareTo(o.getName().toString());
```

Comment:
Maybe you can also make `Name` also implement comparable?
I think It will be easier if we want to compare name again somewhere else.